src/modules/Builder/features/ActivityItems/ActivityItems.utils.ts

```diff
@@ -44,3 +45,21 @@ export const getConditionsToRemove = (
     return !rightSlice[itemKey];
   });
 };
+
+export const getItemsWithVariablesToRemove = (
+  itemName: ItemFormValues['name'],
+  items: ItemFormValues[],
+) => {
+  if (!itemName)
+    return {
+      string: '',
+      list: [],
+    };
+
+  const ItemNamesIncludeSkippableItem = getItemNamesIncludeSkippableItem(itemName, items);
+
+  return {
+    string: ItemNamesIncludeSkippableItem.map((item) => item.name).join(', '),
+    list: ItemNamesIncludeSkippableItem,
```

Comment:
could you give more specific variable names? for example { items, itemsName } smth like that